build/entrypoint.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 
 echo "Building ${RAILS_ENV}"
+sudo chown -R app:app /data/log
+sudo chown -R app:app /usr/local/bundle
```

Comment:
This should be in the image definition I think, so that we don't need sudo, and the image is more self-contained.  The production image will already have the gems installed, and if this is a dev-specific thing, we need to find an approach that doesn't compromise production.